samples/structs.go

```diff
@@ -0,0 +1,31 @@
+// Structs
+package main
+
+import "fmt"
+
+// This `person` struct type has `name` and `good` fields.
```

Comment:
Does it? Does it really? no good. it's malicious.

Fixed: https://goplay.space/#eoBLEu8lRvl

Reply:
:)
Done.